packages/manager/modules/web-paas/src/web-paas.routing.js

```diff
@@ -0,0 +1,45 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('web-paas', {
```

Comment:
```suggestion
  $stateProvider.state('web-paas.index', {
```
To avoid loading the list if we access to a project directly 
Therefore we should have a `web-paas` state that redirectsTo this `.index`